Seccomp error when building tutorial assignments on Debian 8

When trying to install the Tutorial course on Debian 8, I got the following error:
```
Wed Jul  5 01:47:34 EDT 2017
INSTALL assignment config: /usr/local/submitty/GIT_CHECKOUT_Tutorial/examples/01_simple_python/config
        destination:       /var/local/submitty/courses/f17/tutorial/build/01_simple_python

MAKE ERROR
logfile: /var/local/submitty/courses/f17/tutorial/build/01_simple_python/log_make_output.txt


Scanning dependencies of target custom_functions
[ 12%] [ 25%] [ 37%] Building CXX object CMakeFiles/custom_functions.dir/usr/local/submitty/src/grading/empty_custom_function.cpp.o
Building CXX object CMakeFiles/custom_functions.dir/usr/local/submitty/src/grading/seccomp_functions.cpp.o
Building CXX object CMakeFiles/custom_functions.dir/json_generated.cpp.o
In file included from /usr/local/submitty/src/grading/seccomp_functions.cpp:43:
/usr/local/submitty/src/grading/system_call_categories.cpp:139:5: error: use of undeclared identifier '__NR_seccomp'
    ALLOW_SYSCALL(seccomp);
    ^
/usr/local/submitty/src/grading/seccomp_functions.cpp:24:47: note: expanded from macro 'ALLOW_SYSCALL'
#define ALLOW_SYSCALL(name)  allow_syscall(sc,SCMP_SYS(name),#name)
                                              ^
/usr/include/seccomp.h:128:23: note: expanded from macro 'SCMP_SYS'
#define SCMP_SYS(x)             (__NR_##x)
                                 ^
<scratch space>:34:1: note: expanded from here
__NR_seccomp
^
1 error generated.
CMakeFiles/custom_functions.dir/build.make:54: recipe for target 'CMakeFiles/custom_functions.dir/usr/local/submitty/src/grading/seccomp_functions.cpp.o' failed
make[2]: *** [CMakeFiles/custom_functions.dir/usr/local/submitty/src/grading/seccomp_functions.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
CMakeFiles/Makefile2:165: recipe for target 'CMakeFiles/custom_functions.dir/all' failed
make[1]: *** [CMakeFiles/custom_functions.dir/all] Error 2
Makefile:76: recipe for target 'all' failed
make: *** [all] Error 2
```
Commenting out line 139 of system_call_categories.cpp (`ALLOW_SYSCALL(seccomp);`) allowed me to then successfully build all homeworks.

--- a/.setup/bin/setup_sample_courses.py
+++ b/.setup/bin/setup_sample_courses.py
@@ -390,7 +390,7 @@ def parse_args():
     parser.add_argument("--users_path", default=os.path.join(SETUP_DATA_PATH, "users"),
                         help="Path to folder that contains .yml files to use for user creation. Defaults to "
                              "../data/users")
-    parser.add_argument("--submission_url", default="http://192.168.56.101/",help="top level url for the website")
+    parser.add_argument("--submission_url", type=str, default="",help="top level url for the website")
     parser.add_argument("--courses_path", default=os.path.join(SETUP_DATA_PATH, "courses"),
                         help="Path to the folder that contains .yml files to use for course creation. Defaults to "
                              "../data/courses")

--- a/.setup/bin/setup_sample_courses.py
+++ b/.setup/bin/setup_sample_courses.py
@@ -163,7 +163,7 @@ def main():
     with open(list_of_courses_file, "w") as courses_file:
         courses_file.write("")
         for course_id in courses.keys():
-            courses_file.write('<a href="http://192.168.56.101/index.php?semester='+get_current_semester()+'&course='+course_id+'">'+course_id+', '+semester+' '+str(today.year)+'</a>')
+            courses_file.write('<a href="'+args.submission_url+'index.php?semester='+get_current_semester()+'&course='+course_id+'">'+course_id+', '+semester+' '+str(today.year)+'</a>')
             courses_file.write('<br />')
 
     for course_id in courses.keys():
@@ -390,6 +390,7 @@ def parse_args():
     parser.add_argument("--users_path", default=os.path.join(SETUP_DATA_PATH, "users"),
                         help="Path to folder that contains .yml files to use for user creation. Defaults to "
                              "../data/users")
+    parser.add_argument("--submission_url", default="http://192.168.56.101/",help="top level url for the website")
     parser.add_argument("--courses_path", default=os.path.join(SETUP_DATA_PATH, "courses"),
                         help="Path to the folder that contains .yml files to use for course creation. Defaults to "
                              "../data/courses")

--- a/.setup/install_system.sh
+++ b/.setup/install_system.sh
@@ -395,7 +395,7 @@ if [[ ${VAGRANT} == 1 ]]; then
     chmod -R 770 ${SUBMITTY_DATA_DIR}/logs/site_errors
 
     # Call helper script that makes the courses and refreshes the database
-    ${SUBMITTY_REPOSITORY}/.setup/bin/setup_sample_courses.py
+    ${SUBMITTY_REPOSITORY}/.setup/bin/setup_sample_courses.py --submission_url ${SUBMISSION_URL}
 
     #################################################################
     # SET CSV FIELDS (for classlist upload data)

--- a/grading/system_call_categories.cpp
+++ b/grading/system_call_categories.cpp
@@ -136,7 +136,9 @@ void allow_system_calls(scmp_filter_ctx sc, const std::set<std::string> &categor
     ALLOW_SYSCALL(getrusage);
     ALLOW_SYSCALL(move_pages);
     ALLOW_SYSCALL(prlimit64);
+#ifdef __NR_seccomp
     ALLOW_SYSCALL(seccomp);
+#endif
     ALLOW_SYSCALL(setrlimit);
     ALLOW_SYSCALL(ugetrlimit);
     ALLOW_SYSCALL(ulimit);

